internal/service/networkfirewall/firewall_resource_policy_data_source_test.go

```diff
@@ -0,0 +1,74 @@
+package networkfirewall_test
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/networkfirewall"
+	sdkacctest "github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+)
+
+func TestAccNetworkFirewallFirewallResourcePolicyDataSource(t *testing.T) {
```

Comment:
Please rename to meet naming standards
```suggestion
func TestAccNetworkFirewallFirewallResourcePolicyDataSource_basic(t *testing.T) {
```